dumper.php

```diff
@@ -126,7 +126,7 @@ public function clipString($value, $size, $position = 0)
             return $value;
         }
         $position = min($position, $length);
-        $start    = ($size/2 > $position ? 0 : $position - $size/2);
+        $start    = (int)($size/2 > $position ? 0 : $position - $size/2);
```

Comment:
is this because / is float division? Probably better to use intdiv() directly instead

Reply:
Yes, `/` will perform a floating point division. `intdiv()` is an option, but was introduced in PHP 7.0 only, which is larger than the current minimum version of both simpletest and HTMLPurifier. While I personally wouldn't object to raising the minimum version, I wanted to make minimally viable changes in case there I a reason why the requirements have not yet been raised.